styles/globals.css

```diff
@@ -3,3 +3,12 @@
 @tailwind utilities;
 
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+
+ul {
+  list-style-type: disc !important;
+  margin-left: 1em;
+}
+
+.underlined {
```

Comment:
There is [underline class](https://tailwindcss.com/docs/text-decoration) in tailwind. Please consider using it.

Reply:
fixed